.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+## Types of changes
```

Comment:
## Code review pre-checks 
- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I have updated the documentation accordingly (if needed)
- [ ] I have added tests to cover my changes (Infrastructure only)
- [ ] All new and existing unit tests passed (`hydra unit-tests`).

@roydahan WUT?

Reply:
Looks good, but maybe need to be even more specific so people won't just mark all.
like:
- I checked sct_config (or something like that)

didn't copy-paste is too general, I don't think we need it if it doesn't require something specific.

Reply:
I added a relevant lable for backport!!

Reply:
@fruch 
- KISS principle is a link
Add: - [ ] I added a relevant `backport` label

Reply:
add those